### ISSUE TITLE
As a user I would like to see a diff of my editor with the remote resource

### ISSUE THREAD

**Issue by adietish** [redhat-developer/intellij-kubernetes#224]
depends on #218 

When the resource on the cluster changed the editor notifies the user about this change: it'll either automatically reload the editor if there are no local changes or it'll offer the user to reload the resource in the local editor. In this case it would be handy to see what exactly changed. `HttpFileEditor` (a subclass for `RemoteFileEditor`) offers a diff in its toolbar. It would be nice to have something similar:
![image](https://user-images.githubusercontent.com/25126/124155449-c9c4ce80-da96-11eb-89d0-31e7dc97b498.png)
